dist/tools/jlink/jlink.sh

```diff
@@ -58,7 +58,12 @@ _JLINK_SERVER=JLinkGDBServer
 _JLINK_IF=SWD
 _JLINK_SPEED=2000
 # default starting address of the devices flash memory
-_FLASH_ADDR=0
+if [ -n "${FLASH_OFFSET}" ]; then
+    _FLASH_ADDR=${FLASH_OFFSET}
+    echo "Flashing at offset ${FLASH_OFFSET}"
+else
+    _FLASH_ADDR=0
+fi
```

Comment:
One more reason to 'globalize' this varialbe, of in other words to introduce a var called `FLASH_ADDR` that use used throughout all CPUs (possible with a default value set to `0`) and get rid of the `JLINK_FLASH_ADDR` variable alltogether